src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -38,8 +38,9 @@
  * LITERAL_IF},  {@link TokenTypes#LITERAL_SWITCH LITERAL_SWITCH},  {@link
  * TokenTypes#LITERAL_SYNCHRONIZED LITERAL_SYNCHRONIZED},  {@link
  * TokenTypes#LITERAL_TRY LITERAL_TRY},  {@link TokenTypes#LITERAL_WHILE
- * LITERAL_WHILE},  {@link TokenTypes#STATIC_INIT STATIC_INIT},
- * {@link TokenTypes#LAMBDA LAMBDA}.
+ * LITERAL_WHILE},  {@link TokenTypes#STATIC_INIT STATIC_INIT}, {@link
+ * TokenTypes#LITERAL_CASE LITERAL_CASE}, {@link TokenTypes#LITERAL_DEFAULT
+ * LITERAL_DEFAULT}, {@link TokenTypes#LAMBDA LAMBDA}.
  * </p>
  *
```

Comment:
The first sentence is really bad. See how it looks in the rendered docs:
http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/blocks/package-summary.html
Please fix it.

Reply:
Could you please explain me what's bad in the first sentence?

Reply:
The first sentence should be short and clear. In this check it should be something like "Checks the placement of left curly braces."

The RightCurlyCheck is a good example.

Reply:
I am ok to skip this update(do not change) in scope of this issue, could be done in separate issue.
To keep this PR only on target update.

Reply:
I changed class doc a little bit. Now it looks similar to the RightCurlyCheck